### DISCUSSION TITLE
Sharing data to models

### DISCUSSION THREAD

**Discussion by OnekO** [loco-rs/loco#940] [Q&A]
Hi,

I want to process some audit data in a model (example: created_by_id, created_by_ip...), and really I don't know which would be the best way to do that. I want to avoid lazy_static, and as I'm using graphql is not easy to modify the request...

Any better ideas?

**Comment by jondot** [comment]
Hi, I'm trying to understand how `lazy_static` is connected to this? can you share some psuedocode of what you're trying to avoid?
